Logger/LogFactory.cs

```diff
@@ -2,10 +2,27 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string FilePath { get; set; } = "";
+        public BaseLogger? CreateLogger(string className)
         {
-            
-            return null;
+            if(FilePath == "")
+            {
+                return null;
+            }
+
+            BaseLogger fileLog = new FileLogger()
```

Comment:
```suggestion
            return new FileLogger()
```
Alternatively, you can return here rather than creating a new variable and returning the line below it.